model/status_board/v1/peer_dependencies_resource.model

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of peer dependencies.
+resource PeerDependencies {
+  // Retrieves the list of peer dependencies.
+  method List {
+    in out Page Integer = 1
+    in out Size Integer = 100
+    in OrderBy String
```

Comment:
Should be `order`.

Reply:
The `orderBy` is a query param that we've already defined in our own openapi.yaml config.